src/ui/page/skills/questions.html

```diff
@@ -15,32 +30,23 @@ <h2 class="heading heading-large ">Question {index+1} of {count}</h2>
                 {:else}
                 <h2 class="heading heading-large ">Question # of #</h2>
                 {/if}
-                <form action="skills/questions/{index}" method="post" class="question-form">
+                <form action="skills/questions/{index}" id="skip" method="post">
+                    <input type="hidden" name="_csrf" value="{_csrf}">
+                </form>
+                <form id="mainForm" action="skills/questions/{index}" method="post" class="question-form">
                     <input type="hidden" name="_csrf" value="{_csrf}">
                     <fieldset class="form-group">
-                        <QuizQuestion question={question} type={type} />
+                        <QuizQuestion question={question} keys={keys} skipped="{skipped}" attemptedAnswers="{answersToQuestionKeys}" multipleAnswers="{multipleAnswers}"/>
                     </fieldset>
                     <div class="dont-know">
-                        <a href="/skills/questions/{index+1}">I'm not sure</a>
+                        <a class="skipQuestion" onclick="$('#skip').submit()">Skip this question</a>
```

Comment:
What happens when JS is turned off?

Reply:
They can click on Continue button, and that will function the same

Reply:
Offcourse, having JS turned off is not ideal and will cause other pages to act weird

Reply:
hmm. hoping that doesn't become an issue later on. resolving discussion now.